message-passing/introduction/src/test/workflows.test.ts

```diff
@@ -0,0 +1,153 @@
+import { TestWorkflowEnvironment } from '@temporalio/testing';
+import { before, describe, it } from 'mocha';
+import * as wo from '@temporalio/worker';
+import * as greetingWorkflow from '../workflows';
+import { Language } from '../workflows';
+import assert from 'assert';
+const taskQueue = 'test' + new Date().toLocaleDateString('en-US');
```

Comment:
That's useless, since you're using a `TestWorkflowEnvironment` anyway.

Reply:
Thanks, fixed (that was copied from other test files in the repo).